CMakeLists.txt

```diff
@@ -187,6 +187,15 @@ if(CMAKE_C_FLAGS_RELWITHDEBINFO MATCHES DNDEBUG)
   string(REPLACE "-DNDEBUG" "" CMAKE_C_FLAGS_RELWITHDEBINFO "${CMAKE_C_FLAGS_RELWITHDEBINFO}")
 endif()
 
+option(ENABLE_IWYU "Run include-what-you-use with the compiler." ON)
```

Comment:
If this fails should/can we show a message like "you might need to add an entry to cmake.config/mapping.imp"

Reply:
Sounds good. I'm not really at the stage where I've gotten it to actually work, but I'll keep it in mind for later.